install.sh

```diff
@@ -163,21 +154,14 @@ if [ ! -f device.json ]; then
     '. | .[$key0]=$value0 | .[$key1]=[]' <<<'{}' > device.json
 fi
 
-for package in $BOOTSTRAP_PACKAGES; do
-  pkgfile="${CREW_PACKAGES_PATH}/${package}.rb"
+# Functions to maintain packages
 
-  [[ "$(sed -n '/binary_sha256/,/}/p' "${pkgfile}")" =~ .*${ARCH}:[[:blank:]]*[\'\"]([^\'\"]*) ]]
-    sha256s+=("${BASH_REMATCH[1]}")
-
-  [[ "$(sed -n '/binary_url/,/}/p' "${pkgfile}")" =~ .*${ARCH}:[[:blank:]]*[\'\"]([^\'\"]*) ]]
-    urls+=("${BASH_REMATCH[1]}")
-done
-
-# functions to maintain packages
 function download_check () {
     cd "$CREW_BREW_DIR"
-    # use cached file if available and caching enabled
+    # Use cached file if available and caching enabled
     if [ -n "$CREW_CACHE_ENABLED" ] && [[ -f "$CREW_CACHE_DIR/${3}" ]] ; then
+      mkdir -p "$CREW_CACHE_DIR"
+      sudo chown -R "$(id -u)":"$(id -g)" "$CREW_CACHE_DIR"
```

Comment:
Please make this 
```
sudo chown -R "$(id -u)":"$(id -g)" "$CREW_CACHE_DIR" || true
```
as when building in a container some of this will fail...